content/docs/pipelines/sdk/python-based-visualizations.md

```diff
@@ -4,174 +4,59 @@ description = "Predefined and arbitrary visualizations of pipeline outputs"
-    must also be updated.
-        
-        ```csv
-        package_name,url_to_package_license,license_name
-        ```
-        * `package_name`: the name of the package on [pypi](https://pypi.org/)
-        * `url_to_package_license`: the url where the license of the package can
-        be downloaded from
-        * `license_name`: name of package license
-        * Examples for all the columns can be found [here](https://github.com/
-        kubeflow/pipelines/blob/master/backend/src/apiserver/visualization/
-        third_party_licenses.csv).
-10. Submit these changes as a PR or build docker image for usage within your
-cluster.
-
-## Using Predefined Visualizations
-
-1. Open the details of a run
-2. Select a component
+1. Open the details of a run.
+2. Select a component.
     * The component that is selected does not matter. But, if you want to
     visualize the output of a specific component, it is easier to do that within
     that component.
-3. Select the **Artifacts** tab
-4. At the top of the tab you should see a card named **Visualization Creator**
+3. Select the **Artifacts** tab.
+4. At the top of the tab you should see a card named **Visualization Creator**.
 5. Within the card, provide a visualization type, a source, and any necessary
-arguments
-    * Any required or optional arguments will be shown as a placeholder
-6. Click **Generate Visualization**
-7. View generated visualization by scrolling down
+arguments.
+    * Any required or optional arguments will be shown as a placeholder.
+6. Click **Generate Visualization**.
+7. View generated visualization by scrolling down.
 
-## Using Arbitrary Visualizations
+## Using arbitrary visualizations
 
-1. Enable arbitrary visualizations within Kubeflow Pipelines
-2. Open the details of a run
-3. Select a component
+1. Enable arbitrary visualizations within Kubeflow Pipelines.
+2. Open the details of a run.
+3. Select a component.
     * The component that is selected does not matter. But, if you want to
     visualize the output of a specific component, it is easier to do that within
     that component.
-4. Select the **Artifacts** tab
-5. At the top of the tab you should see a card named **Visualization Creator**
+4. Select the **Artifacts** tab.
+5. At the top of the tab you should see a card named **Visualization Creator**.
 6. Within the card, select the **CUSTOM** visualization type then provide a
 source, and any necessary arguments (the source and argument variables are
-optional for custom visualizations)
-7. Provide the arbitrary visualization code
-8. Click **Generate Visualization**
-9. View generated visualization by scrolling down
\ No newline at end of file
+optional for custom visualizations).
+7. Provide the arbitrary visualization code.
+8. Click **Generate Visualization**.
+9. View generated visualization by scrolling down.
+
+## Next Steps
+* Create a [predefined visualization](https://github.com/kubeflow/pipelines/
```

Comment:
Thanks, this is great! It'd be good to add a bit more context, so that Kubeflow users know it's a Kubeflow development task to add a predefined visualization. Something like this:

```
If you'd like to add a predefined visualization to Kubeflow, take a look at the [developer docs](https://github.com/kubeflow/pipelines/blob/master/backend/src/apiserver/visualization/developer_guide.md).
```

(I've also reformatted this as a paragraph instead of a list, as there's currently only one list item.